### ISSUE TITLE
Behavior score bug

### ISSUE THREAD

**Issue by sievenn** [ValveSoftware/Dota2-Gameplay#6488]
### Description

My behavior score was 10000. And in the last three matches, I noticed that my teammates are muted (low priority). And my behaviour score suddenly dropped from 10000 to 61(!). After it, my teammates was toxic. I think its bag.

I've attached all the screenshots below.

### Example Match ID (and possibly Timestamp)

6960198348 - 18:10 (GMT +2)

### Screenshots

![image](https://user-images.githubusercontent.com/122231489/211216106-6049cd31-37ed-4ade-a483-6b1cb3642808.png)

![image](https://user-images.githubusercontent.com/122231489/211216026-ee2a3e5a-094c-4b15-a445-713d7caaf54a.png)

![image](https://user-images.githubusercontent.com/122231489/211216042-26500c72-8ca9-4570-b285-81edacd43ed8.png)

**Comment by jeaz30** [comment]
Да чел, у меня тоже самое, читай ниже. Но они не собираются решать эту проблему....

**Comment by blahoram** [comment]
То же самое, читайте выше. Всем вообще без разницы что кто то ищет игры по 40 минут и играет с руинерами не заслуженно(((

**Comment by jeaz30** [comment]
[#6477](https://github.com/ValveSoftware/Dota2-Gameplay/issues/6477)

**Comment by GauravDe-git** [comment]
+1 , My Friend got same issue

**Comment by sievenn** [comment]
i really hope that supports will do something with it, because i cant play with teammates on this behaivour score

**Comment by jeaz30** [comment]
@jeffhill can you help with this question , because more people got it wrong because of a bug

**Comment by MemoriesOfManaLeak** [comment]
> @jeffhill can you help with this question , because more people got it wrong because of a bug

Some of the reports clearly being from the same person on different account makes this seem like an update to alt detection, no ta bug.

***

OP, you and your alts got caught by the system they use to detect low behavior score alts and match their behavior scores.

It's quite telling that these 2 posts of "different" accounts use the EXACT SAME wording

> My behavior score was 10000. And in the last three matches, I noticed that my teammates are muted (low priority). And my behaviour score suddenly dropped from 10000 to 3147. After it, my teammates was toxic. I think its bag.

#6489

>My behavior score was 10000. And in the last three matches, I noticed that my teammates are muted (low priority). And my behaviour score suddenly dropped from 10000 to 61(!). After it, my teammates was toxic. I think its bag.

#6488 

There's also this one, that also "noticed" the same way you did on these two you reported.

>My behavior score was 9500.Played a few matches noticed i got mostly muted and wierd players.Then my conduct summary came and it showed my behavior score is 1000.I got 0 abandons and less than 3 reports.I play on only one account and never used any cheats. What to do?

#6476

**Comment by sievenn** [comment]
@MemoriesOfManaLeak 

If I understand you correctly, do you think that all these complaints are written by one person?

I live in Eastern Europe, my English is poor.

I use a translator to formulate sentences.

I can provide all IP details.

It's just a coincidence that so many people have posted about this bug, myself included.

**Comment by jeaz30** [comment]
> This is not a bug, this is the system working as intended to catch your attempts to dodge your behavior score. People need to stop falsely reporting it as a bug when it's obvious they're using alts (in dota2 and here on github) in case there actually were false positives that need to get found.

Yes, I did, I ran into this bug. I came here to write about it and saw a similar message already, and decided to just copy the base. Because mine is 100% the same. My decency dropped sharply from 10k to 1k

**Comment by jeaz30** [comment]
> @MemoriesOfManaLeak
> 
> If I understand you correctly, do you think that all these complaints are written by one person?
> 
> I live in Eastern Europe, my English is poor.
> 
> I use a translator to formulate sentences.
> 
> I can provide all IP details.
> 
> It's just a coincidence that so many people have posted about this bug, myself included.          ps. And please, say me, how my behaivour score can dropped from 10k to 61. Not 3k and same
> 
> ps2. i want to answer on your text -
> 
> > It's quite telling that these 2 posts of "different" accounts use the EXACT SAME wording
> 
> If you somehow did not notice that my topic was written earlier, and you expose it as evidence, this is very strange. What do I care if a person simply copied my text? Maybe he speaks English just as poorly, and in order not to bother, he copied the text?

Забавно наблюдать всю эту ситуацию) Они реально думают, что это пишет один и тот же человек. Лол, тут уже более 5 обращений по этому поводу, но поддержка доты 2 просто игнорит это))00

**Comment by OmegaLite** [comment]
да, такая же проблема + у каждого название новое

**Comment by blahoram** [comment]
Такое ощущение что быстрее будет уже новый аккаунт создать и откалибровать, чем ждать ответа и терпеть с руинерами в играх.
